components/inputs/input-select-styles.js

```diff
@@ -95,7 +89,7 @@ export const selectStyles = css`
 
 		.d2l-input-select[aria-invalid="true"] {
 			background-image: ${invalidIcon};
-			background-position: center right calc(1px + 11px + 17px);
+			background-position: center var(--d2l-mirror-end, right) calc(1px + 11px + 17px);
```

Comment:
Is there any chance any of the [flow relative values](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values#data_types_and_values) work?

Reply:
Good idea, but alas they do not.